tests/robustness/crash_consistency_test/crash_consistency_test.go

```diff
@@ -0,0 +1,262 @@
+//go:build darwin || (linux && amd64)
+// +build darwin linux,amd64
+
+package consistency
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/kopia/kopia/tests/recovery/blobmanipulator"
+	"github.com/kopia/kopia/tests/tools/kopiarunner"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConsistencyWhenKill9AfterModify(t *testing.T) {
+	// create, connect repository
+	dataRepoPath := path.Join(*repoPathPrefix, dirPath+dataPath)
+	baseDir := makeBaseDir()
+	bm, err := blobmanipulator.NewBlobManipulator(baseDir, dataRepoPath)
+	if err != nil {
+		if errors.Is(err, kopiarunner.ErrExeVariableNotSet) {
+			log.Println("Skipping recovery tests because KOPIA_EXE is not set")
+		} else {
+			log.Println("Error creating Blob Manipulator:", err)
+		}
+		os.Exit(0)
+	}
```

Comment:
There is a nice error checking interface from `github.com/stretchr/testify/require` package which you can import and use to make this error handling a bit simpler:

```suggestion
	if errors.Is(err, kopiarunner.ErrExeVariableNotSet) {
		t.Skip("Skipping recovery tests because KOPIA_EXE is not set")
	}

	require.NoError(t, err)
```